src/main/java/ca/bc/gov/backendstartapi/repository/VegetationCodeJpaRepository.java

```diff
@@ -0,0 +1,9 @@
+package ca.bc.gov.backendstartapi.repository;
+
+import ca.bc.gov.backendstartapi.entity.VegetationCode;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+/** A {@link JpaRepository} for {@link VegetationCode}. */
+@Repository
+public interface VegetationCodeJpaRepository extends JpaRepository<VegetationCode, String> {}
```

Comment:
I believe a class that extends from JpaRepository don't need to be annotated with `@Repository`.

Reply:
It doesn't, indeed.